### PR TITLE
Adds prettier user-facing aircraft names.

### PR DESCRIPTION
**Pull request by Malakhit**
This makes the names of the aircraft displayed to the player in the UI more verbose and readable.

It allows allows specific countries to display an aircraft's name differently. An example of this would be the JF-17 Thunder, which is known in China as the FC-1 Fierce Dragon - this now displays correctly in the Liberation UI.

### CODE REVIEW

**Comment by Malakhit** [comment]
image
An example of this working.

**Comment by Malakhit** [comment]
image
This also works for the mission package editor.

**Comment by Malakhit** [comment]
image
And in the ATO flights screen!

File: gen/flights/flight.py [hunk 1]
```diff
@@ -175,7 +176,7 @@ def points(self) -> List[FlightWaypoint]:
         return self.flight_plan.waypoints[1:]
 
     def __repr__(self):
-        name = db.unit_type_name(self.unit_type)
+        name = db.unit_pretty_name(self.country, self.unit_type)
```
Comment (DanAlbert) [inline gen/flights/flight.py:185]: `__repr__` should use the ID. It's the debug representation. `__str__` is for pretty printing.

File: qt_ui/windows/basemenu/QRecruitBehaviour.py [hunk 2]
```diff
@@ -58,7 +58,8 @@ def add_purchase_row(self, unit_type: Type[UnitType], layout: QLayout,
         existing_units = self.cp.base.total_units_of_type(unit_type)
         scheduled_units = self.pending_deliveries.units.get(unit_type, 0)
 
-        unitName = QLabel("<b>" + db.unit_type_name_2(unit_type) + "</b>")
+        #unitName = QLabel("<b>" + db.unit_type_name_2(unit_type) + "</b>")
```
Comment (DanAlbert) [inline qt_ui/windows/basemenu/QRecruitBehaviour.py]: Just delete commented out code. We have version control :)

**Review by DanAlbert — CHANGES_REQUESTED** [review]

File: gen/flights/flight.py (hunk 1 above)
Comment (Malakhit) [reply gen/flights/flight.py:185]: Ah, good to know - thanks! Fixed.

File: qt_ui/windows/basemenu/QRecruitBehaviour.py (hunk 2 above)
Comment (Malakhit) [reply qt_ui/windows/basemenu/QRecruitBehaviour.py]: Didn't even spot that one. Thanks, fixed.

**Review by DanAlbert — APPROVED** [review]

**Pull request merged**